Mlem/Views/Tabs/Feeds/Components/FeedHeaderView.swift

```diff
@@ -8,27 +8,23 @@
 import Foundation
 import SwiftUI
 
+protocol FeedType {
+    var label: String { get }
+    var subtitle: String { get }
+    var color: Color? { get }
+    var iconNameFill: String { get }
+    var iconScaleFactor: CGFloat { get }
+}
+
 struct FeedHeaderView: View {
     @EnvironmentObject var appState: AppState
     
-    let feedType: FeedType
+    let feedType: any FeedType
+    let suppressDropdownIndicator: Bool
```

Comment:
Maybe this makes more sense as `showDropdownIndicator` with a default value of `true`?

Reply:
Sure, that works. I think "override default behavior" parameters defaulting to `false` makes more semantic sense, but "show element" parameters defaulting to `true` does save a few `!`s--I'm not strongly opinionated on this one.